tools/nginx/static/nginx.conf

```diff
@@ -0,0 +1,55 @@
+# Basic nginx config for a static site serving static assets.
+
+# This file is mostly taken from the defaults in the official nginx
+# docker image with a few tweaks as commented below.
+
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid  /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include  /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile  on;
+
+    keepalive_timeout  65;
+
+    gzip  on;
+
+    server {
+        listen  80;
+        server_name  localhost;
+
+        location / {
+            root  /usr/share/nginx/html;
+            index  index.html;
+
+            # Set a reasonble default cache time.
+            expires  5m;
+            add_header  Cache-Control "public";
+
+            # Disable caching based on last modified headers as
+            # with our docker builds file timestamps are always the same.
+            add_header Last-Modified "";
```

Comment:
FWIW it might be fairly easy to populate this with a meaningful timestamp.

- bazel populates bazel-out/volatile-status.txt with a `BUILD_TIMESTAMP` variable
- we already have your `expand_templates` BUILD rule

we could do something like:
```
expand_templates(
    ...
    substitutions = {
        "${LAST_MODIFIED_TIMESTAMP}": "{BUILD_TIMESTAMP}",
    },
    stamp = True,
    ...
)
```

Reply:
I did consider this - but then realized this is in the open source repo and we don't have those env vars available here :(

We can do this once we get the docs site building in the closed repo via bazel though!

Reply:
That env variable is in bazel by default, without any changes. (that said, open-source repo doesn't have the relevant expand_templates BUILD rule... yet)